### ISSUE TITLE
Wrap with <React.Fragment>, if templates returns multiple elements

### ISSUE THREAD

**Issue by yuheiy** [pugjs/babel-plugin-transform-react-pug#26]
source:

```js
import React from "react";

class MyComponent extends React.Component {
  render() {
    return pug`
      div foo
      | bar
      div baz
    `;
  }
}
```

compile to:

```js
import React from "react";

class MyComponent extends React.Component {
  render() {
    return React.createElement(
      React.Fragment,
      null,
      React.createElement(
        "div",
        null,
        "foo"
      ),
      "bar",
      React.createElement(
        "div",
        null,
        "baz"
      )
    );
  }
}
```

**Comment by yuheiy** [comment]
We can also write like this:

```js
import React from "react";

class MyComponent extends React.Component {
  render() {
    return pug`
      React.Fragment
        div foo
        | bar
        div baz
    `;
  }
}
```

I thought that should not be implicitly wrapped with `<React.Fragment>`. Sorry.

**Comment by ForbesLindesay** [comment]
We should do this automatically for people.

**Comment by pjungwir** [comment]
@yuheiy have you tested your suggestion? I get this error, because of the period:

    Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.

(See this [similar issue with haml-jsx-loader](https://github.com/dingbat/haml-jsx-loader/issues/2).)

I don't mind wrapping everything in `React.Fragment` by hand, but of course only if it works. :-)

This does work for me:

    return <React.Fragment>
      {pug`
        .fixed-toolbar hi
        .container.under-toolbar
      `}
    </React.Fragment>;

But then I get a loud warning with a stack trace:

    Warning: Each child in an array or iterator should have a unique "key" prop.

**Comment by ezhlobo** [comment]
I'm reopening this issue until we bump new version.

**Comment by ForbesLindesay** [comment]
Released as 6.0.0
